utils/json/src/address.rs

```diff
@@ -119,34 +119,21 @@ pub mod json {
 
 #[cfg(test)]
 mod tests {
+    use forest_shim::address::{Address_v3, Payload};
     use quickcheck_macros::quickcheck;
     use serde_json;
 
     use super::*;
 
-    #[cfg(test)]
-    #[derive(Clone, Debug, PartialEq)]
-    struct AddressWrapper {
-        address: Address,
-    }
-
-    #[cfg(test)]
-    impl quickcheck::Arbitrary for AddressWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let address: fvm_shared::address::Address = arbitrary::Arbitrary::arbitrary(
-                &mut arbitrary::Unstructured::new(&Vec::arbitrary(g)),
-            )
-            .unwrap();
-            AddressWrapper {
-                address: Address::from(address),
-            }
-        }
-    }
-
     #[quickcheck]
-    fn address_roundtrip(address: AddressWrapper) {
-        let serialized = serde_json::to_string(&json::AddressJsonRef(&address.address)).unwrap();
-        let parsed: json::AddressJson = serde_json::from_str(&serialized).unwrap();
-        assert_eq!(address.address, parsed.0);
+    fn address_roundtrip(address: Address_v3) {
+        let address = address.into();
```

Comment:
Let's implement `Arbitrary` for `Address`. Then we can just write this:

```suggestion
    fn address_roundtrip(address: Address) {
```